distributor-node/config.yml

```diff
@@ -19,6 +19,9 @@ logs:
   #   auth:
   #     username: username
   #     password: password
+# otlp:
```

Comment:
This needs to be uncommented as the properties are "required".
Otherwise change them to be optional and leave them commented out.
If node is started with instrumentation and endpoint/attributes not provided, exist with error?

Reply:
Actually, the `otlp` properties are not required in the config schema. You can see [here](https://github.com/zeeshanakram3/joystream/blob/8c0494ede006d155f91dad8e3424efd33f0918a9/distributor-node/src/schemas/configSchema.ts#L20). Also, the `distributor-1` service works fine if I remove the `JOYSTREAM_DISTRIBUTOR__OTLP__ENDPOINT`, `JOYSTREAM_DISTRIBUTOR__OTLP__ATTRIBUTES` env vars from the service in docker-compose.yml file, which also enforces the point that `otlp` properties are not required. I think there is a problem with the `ValidationService` class. I tried to debug the problem by adding `console.log(this.avj.errors)` at [line](https://github.com/zeeshanakram3/joystream/blob/8c0494ede006d155f91dad8e3424efd33f0918a9/distributor-node/src/services/validation/ValidationService.ts#L34), and got this:
```
[
  {
    keyword: 'required',
    dataPath: '/otlp',
    schemaPath: '#/properties/otlp/required',
    params: { missingProperty: 'attributes' },
    message: "should have required property 'attributes'"
  }
]
/joystream/distributor-node/lib/services/parsers/ConfigParserService.js:97
                    throw e;
                    ^

ValidationError: Invalid env value of JOYSTREAM_DISTRIBUTOR__OTLP__ENDPOINT
```

Reply:
Just created an issue for this. Will be investigating it https://github.com/Joystream/joystream/issues/4787

Reply:
okay, so how do we interpret the line `required: ['endpoint', 'attributes'],` in [configSchema.ts](https://github.com/zeeshanakram3/joystream/blob/8c0494ede006d155f91dad8e3424efd33f0918a9/distributor-node/src/schemas/configSchema.ts#L131)

Reply:
> okay, so how do we interpret the line `required: ['endpoint', 'attributes'],` in [configSchema.ts](https://github.com/zeeshanakram3/joystream/blob/8c0494ede006d155f91dad8e3424efd33f0918a9/distributor-node/src/schemas/configSchema.ts#L131)

This means `otlp` is an optional object in configSchema. However, when it is present (i.e., not `undefined`),  it `must` have `endpoint` & `attributes` properties. 

FYI I created a PR to address this issue, please have a look #4788

Reply:
Merged the fix, perhaps you want to update this from master

Reply:
Done.